aweber.com.email-authentication-optin.json

```diff
@@ -0,0 +1,35 @@
+{
+   "providerId":"aweber.com",
+   "providerName":"AWeber",
+   "serviceId":"email-authentication-optin",
+   "serviceName":"AWeber Email Authentication for Optin.com",
+   "version": 1,
+   "logoUrl":"https://assets.aweber-static.com/page-templates/assets/img/powered_by.png",
+   "description":"Configure AWeber's SPF and DKIM records to improve sender reputation and deliverability.",
+   "variableDescription":"{%subdomain%} is the AWeber customer's unique subdomain",
+   "records":[
+      {
+         "type": "SPFM",
+         "host": "@",
+         "spfRules": "include:{%account-subdomain%}.optin.com"
```

Comment:
Please also remove curly brackets from the variable syntax. It's only `%variablename%`

Reply:
ty done.